### PR TITLE
SONARHTML-276 Remove dev from owners

### PR DESCRIPTION
**Pull request by kebetsi**
[SONARHTML-276](https://sonarsource.atlassian.net/browse/SONARHTML-276)

The logical changes are in the `pom.xml` files.

1. Remove Galesloot from owners and javadoc.
2. Update remaining LGPL licenses to SSAL

The license setup is still flacky, but is out of scope for now: [SONARHTML-260](https://sonarsource.atlassian.net/jira/software/c/projects/SONARHTML/issues/SONARHTML-260)

[SONARHTML-276]: https://sonarsource.atlassian.net/browse/SONARHTML-276?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SONARHTML-260]: https://sonarsource.atlassian.net/browse/SONARHTML-260?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: its/plugin/src/test/java/com/sonar/it/web/FileSuffixesTest.java [hunk 1]
```diff
@@ -1,19 +1,18 @@
 /*
- * SonarSource :: HTML :: ITs :: Plugin
- * Copyright (c) 2011-2024 SonarSource SA and Matthijs Galesloot
- * sonarqube@googlegroups.com
+ * SonarQube HTML
```
Comment (chrislain-razafimahefa-sonarsource) [inline its/plugin/src/test/java/com/sonar/it/web/FileSuffixesTest.java:2]: Why is the name 'SonarQube' and not 'SonarSource' ?

File: its/plugin/src/test/java/com/sonar/it/web/FileSuffixesTest.java [hunk 2]
```diff
@@ -1,19 +1,18 @@
 /*
- * SonarSource :: HTML :: ITs :: Plugin
- * Copyright (c) 2011-2024 SonarSource SA and Matthijs Galesloot
```
Comment (chrislain-razafimahefa-sonarsource) [inline its/plugin/src/test/java/com/sonar/it/web/FileSuffixesTest.java:3]: Is it ok to remove the guy's name from the license if he was there before?

File: its/plugin/src/test/java/com/sonar/it/web/FileSuffixesTest.java (hunk 1 above)
Comment (kebetsi) [reply its/plugin/src/test/java/com/sonar/it/web/FileSuffixesTest.java:2]: I aligned to what is done in sonar-java: https://github.com/SonarSource/sonar-java/blob/master/sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaPlugin.java

File: its/plugin/src/test/java/com/sonar/it/web/FileSuffixesTest.java (hunk 2 above)
Comment (kebetsi) [reply its/plugin/src/test/java/com/sonar/it/web/FileSuffixesTest.java:3]: We had a long discussion by email and slack with Nico Peru about it. We got his approval and this PR is the execution.

**Pull request merged**